src/DolbyIO.Comms.Sdk/Services/MediaDeviceService.cs

```diff
@@ -15,48 +15,93 @@ namespace DolbyIO.Comms.Services
     /// </summary>
     public sealed class MediaDeviceService
     {
-        private DeviceAddedEventHandler _added;
+        private AudioDeviceAddedEventHandler _audioAdded;
 
         /// <summary>
-        /// Sets the <see cref="DeviceAddedEventHandler"/> that is raised when a new audio or video device is added to the system.
+        /// Sets the <see cref="AudioDeviceAddedEventHandler"/> that is raised when a new audio device is added to the system.
         /// </summary>
-        /// <value>The <see cref="DeviceAddedEventHandler"/> event handler.</value>
-        public DeviceAddedEventHandler Added
+        /// <value>The <see cref="AudioDeviceAddedEventHandler"/> event handler.</value>
+        public AudioDeviceAddedEventHandler AudioAdded
```

Comment:
Could we use `AudioDeviceAdded`, without `Device` it sounds like you are getting an audio stream.

Reply:
The code will look like thise `_sdk.MediaDevice.AudioAdded = () => {}`. When you read it, it is pretty straightforward. Adding Device will look like: `_sdk.MediaDevice.AudioDeviceAdded = () => {}` adding a repeating Device word. What do you think ?

Reply:
Yes, I like it better (just my opinion), we can ask others what they think. This is consistent with our Web SDK: https://docs.dolby.io/communications-apis/docs/js-client-sdk-mediadeviceservice